lib/fawry/requests/charge_request.rb

```diff
@@ -5,7 +5,7 @@
 module Fawry
   module Requests
     module ChargeRequest
-      DEFAULTS = { payment_method: 'PAYATFAWRY', currency_code: 'EGP' }.freeze
+      DEFAULTS = { payment_method: 'PAYATFAWRY', currency_code: 'EGP', language: 'en-gb' }.freeze
```

Comment:
I think it's better to have it as `'ar-eg'` 'cause most people want to get the notification in Arabic. WDYT?

Reply:
Makes sense

Reply:
updated!